Framework/Core/Windows/Devices/I2c/I2cDevice.cs

```diff
@@ -0,0 +1,206 @@
+using System;
+using Microsoft.SPOT.Hardware;
+
+namespace Windows.Devices.I2c
+{
+    public struct I2cTransferResult
+    {
+        public I2cTransferStatus Status;
+        public uint BytesTransferred;
+    }
+
+    public sealed class I2cDevice : IDisposable
+    {
+        // Construction and destruction
+
+        internal I2cDevice(ushort slaveAddress, I2cBusSpeed busSpeed)
+        {
+            int clockRateKhz = 100;

```

Comment:
The entire purpose for this variable is to give these values a name (clockRate) and a well-defined unit of measurement (khz).